card/mod/05_standard/set/type/search_type.rb

```diff
@@ -63,25 +63,34 @@ def get_query params={}
   end
 
   view :card_list do |args|
-    if search_vars[:results].empty?
+    if search_results.empty?
       'no results'
     else
-      search_vars[:results].map do |c|
+      search_results.map do |c|
         nest c
       end.join "\n"
     end
   end
   
   def search_vars args={}
-    @search_vars ||= begin
-      v = {}
-      v[:query] = card.query search_params
-      v[:item] = set_inclusion_opts args.merge( :query_view=>v[:query][:view] )
-      v[:results]  = card.item_cards search_params  # this is really odd.  the search is called from within the vars???
-      v
-    rescue =>e
-      { :error => e }
-    end
+    @search_vars ||=
+      begin
+        v = {}
+        v[:query] = card.query(search_params)
+        v[:item]  = set_inclusion_opts args.merge( :query_view=>v[:query][:view] )
+        v
+      rescue =>e
```

Comment:
unlikely.

I *think* that the reason we did things this way was that the old cloning-based format approached every instance variable had to be clearer by hand.  Since that no longer applies, I'd guess this is safe.